### DISCUSSION TITLE
CurseForge app is not happy

### DISCUSSION THREAD

**Discussion by ModernGamingWorld** [cc-tweaked/CC-Tweaked#2017] [General]
<img width="1503" alt="Screenshot 2024-11-22 123443" src="https://github.com/user-attachments/assets/a9d207b0-3bc9-45d0-bbd2-8fcbd134b560">

The CurseForge app is not happy because you archived the version cc-tweaked-1.20.1-forge.1.114.0.jar. This happened right after I pulled the version from Modrinth since you chose to post your versions only on Modrinth. This, however, is going to cause major issues because not everyone uses Modrinth, and I myself don't check for updates for mods on there as I use CurseForge for my main workflow. So id suggest not removing them from Curseforge, as your just causing more problems for yourself.

**Comment by SquidDev** [comment]
> The CurseForge app is not happy because you archived the version cc-tweaked-1.20.1-forge.1.114.0.jar. 

Ahh, apologies if that has confused the app. 1.114 has a pretty major bug (#2010), and it didn't feel fair to folks to leave that as the latest/default version.

> So id suggest not removing them from Curseforge, as your just causing more problems for yourself.

I'm aware this is going to inconvenience some people, and I can only apologise for that. I've been wanting to do this for a while now, but putting it off because I know it is a pain. However, I no longer wish to support CurseForge/Overwolf, and so will not be publishing my stuff there.

**Reply by ModernGamingWorld** [reply]
Well, Curseforge is denying mine and other people's server files because of you removing files. I have sent a message to Curseforge to fix it or reach out to you

**Reply by fatboychummy** [reply]
You should just be able to update your server file to the previous stable version, no?

**Reply by SquidDev** [reply]
Just to clarify, the files have been archived rather than outright removed. This *should* allow modpacks to continue to use the file, it just won't show up for people looking to download the mod manually. If that's not working, then that sounds like a bug with the Curse app.

**Reply by RainerZufall187** [reply]
ok so whats wrong with curseforge?

**Reply by NovaLynxie** [reply]
> ok so whats wrong with curseforge?

doubt this will be seen anymore and wasn't answered, so guessing it had something to do with how things were going on overwolf's side amongst some other details (too much to really specify, you'd be better off looking into this yourself).
am hoping at least modrinth becomes more viable for more mods as time goes on since, I'll admit this is or was a pain to work around.
